test/blocks/global-navigation/global-navigation.test.js

```diff
@@ -227,15 +227,6 @@ describe('global navigation', () => {
       });
       promoMeta.remove();
     });
-
-    it('doesn\'t exist on mobile', async () => {
```

Comment:
Instead of removing it, how about we write a test that the promo exists on mobile?

Reply:
Added unit tests for both cases: when it exists and when it does not exist on mobile.